pressurecooker/youtube.py

```diff
@@ -26,32 +35,95 @@ def __init__(self, url):
 
         :param url: URL of a YouTube resource. URL may point to a video, playlist or channel.
         """
+
+        if not 'youtube.com' in url:
```

Comment:
Not sure if it's likely to happen, but user could add a url of the form `https://youtu.be/{{youtube_id}}` which is the same as `https://www.youtube.com/watch?v={{youtube_id}}`